### PR TITLE
Rename `remote` to `forge`

### PR DESCRIPTION
**Pull request by qwerty287**
As of #745

### CODE REVIEW

File: cli/exec/flags.go [hunk 1]
```diff
@@ -123,8 +123,8 @@ var flags = []cli.Flag{
 		Name:    "repo-link",
 	},
 	&cli.StringFlag{
-		EnvVars: []string{"CI_REPO_REMOTE"},
-		Name:    "repo-remote-url",
+		EnvVars: []string{"CI_REPO_CLONE_URL"},
```
Comment (6543) [inline cli/exec/flags.go:126]:
```suggestion
		EnvVars: []string{"CI_REPO_CLONE_URL", "CI_REPO_REMOTE"},
```

**Comment by 6543** [comment]
just a question: as "remote" never did pop up any questions, do we need to rename all "remote" stuff as forge?

I mean we could have support for non-forge code hosts in future and so the name would be wrong ...

In my opinion we just have to address the name "forge" in the docs ... if the remote is a forge (github, gitlab, gitea, ...)
but can just all it remote in code

**Comment by anbraten** [comment]
As @qwerty287 said adding non-forge code systems is very unlikely a thing being added in the near future considering the amount of work, but I guess  even if someone is doing that it wouldn't still be to problematic to have that system called forge as well. In general I am mainly interested in consistency whatever the term would be.

File: cli/exec/flags.go [hunk 2]
```diff
@@ -123,8 +123,8 @@ var flags = []cli.Flag{
 		Name:    "repo-link",
 	},
 	&cli.StringFlag{
-		EnvVars: []string{"CI_REPO_REMOTE"},
-		Name:    "repo-remote-url",
+		EnvVars: []string{"CI_REPO_CLONE_URL", "CI_REPO_REMOTE"},
+		Name:    "repo-clone-url",
```
[the remark this answers is not in the document]
Comment (anbraten) [reply cli/exec/flags.go:127]: https://github.com/woodpecker-ci/woodpecker/issues/276

**Review by anbraten — APPROVED** [review]

**Comment by 6543** [comment]
well no compains from my side

**Comment by 6543** [comment]
Lint error

File: server/api/cron.go [hunk 3]
```diff
@@ -150,8 +150,9 @@ func PatchCron(c *gin.Context) {
 		return
 	}
 	if in.Branch != "" {
-		// check if branch exists on remote
-		_, err := remote.BranchHead(c, user, repo, in.Branch)
+		// check if branch exists on forge
+
```
Comment (6543) [inline server/api/cron.go:154]:
```suggestion
```

**Review by 6543 — APPROVED** [review]

**Pull request merged**